api/controllers/PackageVersionController.js

```diff
@@ -99,7 +99,7 @@ module.exports = {
     })
     // The method above will be cached
     .then(function(version){
-      if(version === null) return res.redirect(301, '/packages/' + encodeURIComponent(packageName));
+      if(version === null) return res.rstudio_redirect(303, '/packages/' + encodeURIComponent(packageName));
```

Comment:
changed this because I had a problem with the ajax request with package 'haven' that had a triple redirect. But the code didn't matter, forgot to change it back.
(also, problem with haven was solved another way)